common/djangoapps/student/migrations/0003_auto_20170824_1606.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('student', '0002_auto_20151208_1034'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='userprofile',
+            name='nonregistered',
+        ),
+        migrations.AlterField(
```

Comment:
I think we can try to drop this;
it seems unrelated.

Reply:
And, to drop this, do we just delete it from the migration file?